src/style.css

```diff
@@ -2,2 +1,90 @@
+@font-face {
+  font-family: Roboto;
+  src: url('/fonts/Roboto-Medium.ttf') format('truetype');
+  font-weight: 500;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider providing a fallback font stack for better compatibility across different browsers and systems. The current stack is good, but adding more fallbacks can improve robustness.